utils/zerofrom/derive/Cargo.toml

```diff
@@ -0,0 +1,31 @@
+# This file is part of ICU4X. For terms of use, please see the file
+# called LICENSE at the top level of the ICU4X source tree
+# (online at: https://github.com/unicode-org/icu4x/blob/main/LICENSE ).
+
+[package]
+name = "zerofrom-derive"
+version = "0.1.0"
+description = "Custom derive for the zerofrom crate"
+repository = "https://github.com/unicode-org/icu4x"
+license-file = "LICENSE"
+categories = ["data-structures", "memory-management", "caching", "no-std"]
+keywords = ["zerocopy", "serialization", "lifetime", "borrow"]
+authors = ["Manish Goregaokar <manishsmail@gmail.com>"]
+edition = "2018"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[lib]
+proc_macro = true
+path = "src/lib.rs"
+
+[dependencies]
+proc-macro2 = "1.0.27"
+quote = "1.0.9"
+syn = { version = "1.0.73", features = ["derive", "fold"] }
+synstructure = "0.12.4"
+
+[dev-dependencies]
+zerofrom = { version = "0.1.0", path = "..", features = ["derive"]}
```

Comment:
Issue: Self-dependency

Reply:
That's not a self-dependency, this is the derive